libraries/UR_Atmosphere/atmosphere_definitions.h

```diff
@@ -0,0 +1,109 @@
+/*
+   Aeronautics Atmosphere Calculations.
+   Copyright (c) 2019 Hiroshi Takey <htakey@gmail.com>
+
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include <cmath>
+
+#define INHG_TO_PA      3386.383613f
+#define METERS_TO_FEET  3.280839895f
+
+namespace ISA_MATH_CONST {
```

Comment:
why aren't these in the other class?

Reply:
ohh ok, i forget it, i will fix it, thanks.